package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "prettier",
-  "version": "1.18.1",
+  "name": "@observablehq/prettier",
```

Comment:
Why?

Reply:
Sorry, I was just trying to look at the diffs in this branch.